vo_models/uws/types.py

```diff
@@ -7,41 +7,63 @@ class ErrorType(str, Enum):
     """Enum for error types."""
 
     TRANSIENT = "transient"
+    """The error is transient and the job may be rerun."""
     FATAL = "fatal"
+    """The error is fatal and the job may not be rerun."""
+
```

Comment:
What do these extra triple quoted strings in the enum do here? Are these for the docs?

Reply:
Yes. I've moved the rest of the projects to Google-style docstrings that are compatible with the Sphinx Napoleon autodoc extension. Using that style in Enum docstrings don't actually seem to show up in the docs, but this does. It give nice option-level doc output. 

Something like this, for example: https://vo-models.readthedocs.io/en/latest/pages/api/uws_api.html#vo_models.uws.types.ExecutionPhase

Reply:
Interesting. I guess I'm just not used to seeing doc strings like this.